bot/exts/utilities/conversationstarters.py

```diff
@@ -56,21 +56,45 @@ def _build_topic_embed(channel_id: int) -> discord.Embed:
             color=discord.Colour.og_blurple()
         )
 
-        try:
-            channel_topics = TOPICS[channel_id]
-        except KeyError:
-            # Channel doesn't have any topics.
-            embed.title = f"**{next(TOPICS['default'])}**"
+        if previous_topic is None:
+            # Message first sent
+            try:
+                channel_topics = TOPICS[channel_id]
+            except KeyError:
+                # Channel doesn't have any topics.
+                embed.title = f"**{next(TOPICS['default'])}**"
+            else:
+                embed.title = f"**{next(channel_topics)}**"
         else:
-            embed.title = f"**{next(channel_topics)}**"
+            # Message is being edited
+            try:
+                channel_topics = TOPICS[channel_id]
+            except KeyError:
+                # Channel doesn't have any topics.
+                new_topic = f"**{next(TOPICS['default'])}**"
+            else:
+                new_topic = f"**{next(channel_topics)}**"
```

Comment:
This is duplicated code, doing the shared logic first and returning early if it's the first message means you don't need to do this twice.
```python
        try:
            channel_topics = TOPICS[channel_id]
        except KeyError:
            # Channel doesn't have any topics.
            embed.title = f"**{next(TOPICS['default'])}**"
        else:
            embed.title = f"**{next(channel_topics)}**"

        if previous_topic is None:
            # This is the first topic being sent
            return embed

        total_topics = previous_topic.count("\n") + 1
        # Add 1 before first topic
        if total_topics == 1:
            previous_topic = f"1. {previous_topic}"

        embed.title = previous_topic + f"\n{total_topics + 1}. {embed.title}"

        # When the embed will be larger than the limit, use the previous embed instead
        if len(embed.title) > 256:
            embed.title = previous_topic

        return embed
```